stylesheets/_header.scss

```diff
@@ -20,7 +20,13 @@
                 height: 1.56em;
                 width: initial;
             }
-        }
+
+            div.f-org-name {
+                position:fixed;
```

Comment:
Position `fixed` is not a great choice here.  That's for forcing layout to ignore scroll position, which might work in this case because those elements won't scroll, but is semantically not what we want.  `position:absolute` might be more appropriate or better yet, we could nest the elements in flex elements.  Flex is preferred but I'd take position absolute if there is schedule pressure (none from me).